ui/src/lib/features/k8s/storage/persistentvolumeclaims/store.ts

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024-Present The UDS Authors
+
+import type { V1PersistentVolumeClaim as Resource } from '@kubernetes/client-node'
+
+import { ResourceStore, transformResource } from '$features/k8s/store'
+import { type ColumnWrapper, type CommonRow, type ResourceStoreInterface } from '$features/k8s/types'
+
+interface Row extends CommonRow {
+  storage_class: string
+  capacity: string
+  pods: string[]
+  status: string
+}
+
+export type Columns = ColumnWrapper<Row>
+
+export function createStore(): ResourceStoreInterface<Resource, Row> {
+  const url = `/api/v1/resources/storage/persistentvolumeclaims`
```

Comment:
```suggestion
  const url = `/api/v1/resources/storage/persistentvolumeclaims?dense=true`
```
missing fields that come from dense

Reply:
agreed this needs the `dense=true` flag now